Modules/Options/GeneralTab/QuestieOptionsGeneral.lua

```diff
@@ -663,19 +683,35 @@ function QuestieOptions.tabs.general:Initialize()
             },
             objectiveCompleteSoundButton = {
                 type = "execute",
-                order = 2.14,
+                order = 2.15,
                 name = "",
                 width = 0.5,
                 image = function ()
                     return "Interface\\OptionsFrame\\VoiceChat-Play", 15, 15
                 end,
                 func = function ()
-                    PlaySoundFile(Sounds.QUEST_OBJECTIVE_COMPLETE_SOUND_FILE)
+                    local soundFile = Questie.db.char.objectiveCompleteSoundChoice ~= nil and Questie.db.char.objectiveCompleteSoundChoice or Sounds.QUEST_COMPLETE_SOUND_FILE
+                    PlaySoundFile(soundFile, "Master")
                 end
             },
+            objectiveCompleteSoundChoice = {
+                type = "select",
+                order = 2.16,
+                values = _GetObjectiveSoundChoices(),
+                sorting = _GetObjectiveSoundChoicesSort(),
+                style = 'dropdown',
+                name = function() return l10n('Objective Complete Sound Selection') end,
+                desc = function() return l10n('The sound you hear when an objective is completed'); end,
```

Comment:
These translations need to be added to https://github.com/Questie/Questie/blob/master/Localization/Translations/Options/General.lua

Reply:
done